vmas/simulator/environment/__init__.py

```diff
@@ -9,13 +9,25 @@
 class Wrapper(Enum):
     RLLIB = 0
     GYM = 1
+    GYMNASIUM = 2
+    GYMNASIUM_VEC = 3
 
-    def get_env(self, env: Environment):
+    def get_env(self, env: Environment, **kwargs):
         if self is self.RLLIB:
             from vmas.simulator.environment.rllib import VectorEnvWrapper
 
-            return VectorEnvWrapper(env)
+            return VectorEnvWrapper(env, **kwargs)
         elif self is self.GYM:
-            from vmas.simulator.environment.gym import GymWrapper
+            from vmas.simulator.environment.gym.gym import GymWrapper
```

Comment:
we need to avoid changing the import structure for bc-compatibility

what we can do to makes sure
`from vmas.simulator.environment.gym import GymWrapper` still runs is:

add `__init__.py` to the `gym` folder containing:

```python
from .gym inport GymWrapper
from .gymnasium import GymnasiumWrapper
...
```

that way `from vmas.simulator.environment.gym import GymWrapper`
still works
and also `from vmas.simulator.environment.gym import GymnasiumWrapper` will work